backend/tracim_backend/lib/search/elasticsearch_search/elasticsearch_search.py

```diff
@@ -739,10 +750,8 @@ def search_workspace(
             search = search.filter("terms", member_ids=member_ids)
         if not show_deleted:
             search = search.exclude("term", is_deleted=True)
-        if size:
-            search = search.extra(size=size)
-        if page_nb:
-            search = search.extra(from_=self.offset_from_pagination(size, page_nb))
+        if size and page_nb:
```

Comment:
Same here, i don't understood this change. It make more sense to have similar code as for content here.